lib/s3middleware/objectRetention.js

```diff
@@ -140,14 +140,14 @@ function parseRetentionXml(xml, log, cb) {
 function convertToXml(retentionInfo) {
     const xml = [];
     const ret = retentionInfo.retention;
-    xml.push('<ObjectRetention xmlns="http://s3.amazonaws.com/doc/2006-03-01/">');
+    xml.push('<Retention xmlns="http://s3.amazonaws.com/doc/2006-03-01/">');
     if (ret && ret.mode && ret.retainUntilDate) {
         xml.push(`<Mode>${ret.mode}</Mode>` +
```

Comment:
Since we are pushing to a list and join, we can push each of these XML attributes separately. It might look cleaner.